std/wasi/snapshot_preview1.ts

```diff
@@ -285,7 +284,7 @@ export default class Context {
   constructor(options: ContextOptions) {
     this.args = options.args ? options.args : [];
     this.env = options.env ? options.env : {};
-    this.memory = options.memory!;
+    this.memory = null!;
```

Comment:
`!` is not needed here?

Reply:
>  memory: WebAssembly.Memory;		

It's actually nullable according to the type rules, it's forced to null here and assigned/validated later in the start method.